.devcontainer/devcontainer.json

```diff
@@ -14,7 +14,10 @@
   ],
   "workspaceMount": "source=${localWorkspaceFolder},target=/ic,type=bind",
   "workspaceFolder": "/ic",
-  "initializeCommand": "mkdir -p ~/.aws ~/.ssh ~/.cache ~/.local/share/fish && touch ~/.zsh_history ~/.bash_history",
+  "initializeCommand": "mkdir -p ~/.aws ~/.ssh ~/.cache/cargo ~/.local/share/fish && touch ~/.zsh_history ~/.bash_history",
+  "containerEnv": {
+    "CARGO_TARGET_DIR": "/home/ubuntu/.cache/cargo"
```

Comment:
IIUC `CARGO_TARGET_DIR` defaults to `target` relative to the current working directory. So if we mount `/hoststorage/cache/cargo` to `/ic/target` below do we need this override of `CARGO_TARGET_DIR`. Note I'm probably just missing some key detail to understanding this.

Reply:
Section `containerEnv` sets `CARGO_TARGET_DIR` to be `/home/ubuntu/.cache/cargo` which overrides the default.

Reply:
Right, this `containerEnv` section overrides it but I was wondering why since I was expecting the default of `target` to be good already since `/ic/target` points to the host's storage.

Note feel free to close this. This is just me trying to understand what's going on.